README.md

```diff
@@ -97,7 +97,7 @@ provides the common behavior for `ConsumerRegistry` implementations. Below is a
 ## Producer overview
 
 The core class of the producer module is `WSRPProducer` and its `WSRPProducerImpl` implementation. The producer is configured using a `ProducerConfigurationService` implementation
-and delegates WSRP calls to a `PortletInvoker`, which can access the portlets that have been deployed and made remotable. The producer manages consumers' registrations using a
+and delegates WSRP calls to a `PortletInvoker`, which can access the portlets that have been deployed and made removable. The producer manages consumers' registrations using a
```

Comment:
That's actually not a typo, it's indeed intended to mean "remote portlets". As "remotable" isn't a word, it could be changed to "have been deployed and made available for remote access".